client/components/BlogPost.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import NextLink from 'next/link'
+import { useColorMode, Heading, Text, Flex, Box, Link } from '@chakra-ui/react'
+import { parseISO, format } from 'date-fns'
+
+const BlogPost = ({ title, publishedAt, summary, slug }) => {
+    const { colorMode } = useColorMode()
+    const secondaryTextColor = {
+        light: 'gray.700',
+        dark: 'gray.400'
+    }
```

Comment:
It might be a good idea to consolidate style values in a central location (e.g. `colors.js` or something) since these values may be used in multiple places

Reply:
Same for styles in other components below

Reply:
Yeah, I was worried about the slow loading times too––was suspecting if the production build could be faster but I always never seem to be able to run prod. Glad you were able to test it in prod build! :)

Reply:
`todo`

Reply:
Personally I think `DarkModeSwitch` may be a good place to hold this, but if you think it's not worth refactoring at this point, that's fine as well